src/test/scala/is/hail/methods/LinearRegressionSuite.scala

```diff
@@ -179,20 +254,14 @@ class LinearRegressionSuite extends SparkSuite {
         root = Some("sa.cov"),
         config = TextTableConfiguration(types = Map("Cov1" -> TDouble, "Cov2" -> TDouble)))
       .annotateSamplesFam("src/test/resources/regressionLinear.fam")
-      .linreg("sa.fam.isCase", Array("sa.cov.Cov1", "sa.cov.Cov2"), "va.linreg", 1, 0.0)
-
-    val v1 = Variant("1", 1, "C", "T")
-    // x = (0, 1, 0, 0, 0, 1)
-    val v2 = Variant("1", 2, "C", "T")
-    // x = (., 2, ., 2, 0, 0)
-    val v6 = Variant("1", 6, "C", "T")
-    // x = (0, 0, 0, 0, 0, 0)
-    val v7 = Variant("1", 7, "C", "T")
-    // x = (1, 1, 1, 1, 1, 1)
-    val v8 = Variant("1", 8, "C", "T")
-    // x = (2, 2, 2, 2, 2, 2)
-    val v9 = Variant("1", 9, "C", "T")
-    // x = (., 1, 1, 1, 1, 1)
+      .linreg("sa.fam.isCase", Array("sa.cov.Cov1", "sa.cov.Cov2"), "va.linreg", useDosages = false, 1, 0.0)
```

Comment:
don't really need to add `useDosages = false` since it's the default case, but happy to have it here.